### ISSUE TITLE
add range routes

### ISSUE THREAD

**Issue by mrhut10** [mrhut10/futuresFineFurniture#45]
currently after #42 was resolved we have a related products section which displays three products from the same range as the product you are looking at.

however a show even more products link should be here which takes you too a page full of products from the same range.

I've starting mapping out what changes I'll need to make to implement.

**Comment by mrhut10** [comment]
made a first attempt at this which should be on staging server now the route is
\collections

**Comment by mrhut10** [comment]
have a branch working with this locally. :)

**Comment by mrhut10** [comment]
this has been resolved and currently testing in staging branch, and planned to be merged into master shortly in pr #51
